src/system/ExtensionsModule/Resources/docs/ExtensionsInterfaceController_and_Twig_tags.md

```diff
@@ -0,0 +1,53 @@
+ExtensionsInterfaceController
```

Comment:
All the docs in this PR should be moved to the CORE docs folder (e.g. `src/docs/Core-2.0/`) please. They will never be found buried in the Extensions or Admin modules ;-)